bin/cassandra.in.sh

```diff
@@ -125,9 +125,6 @@ short=$(echo "${jvmver}" | cut -c1-2)
 JAVA_VERSION=17
 if [ "$short" = "11" ]  ; then
      JAVA_VERSION=11
-elif [ "$JVM_VERSION" \< "17" ] ; then
```

Comment:
are there any existing checks/conditions for when JVM_VERSION is greater than 17 ? 

instead of introducing a flag, i'd be ok with 
- preventing JDKs 12-16
- printing a loud warning when 18+ is used

I'm ~ok when extending the latter to failure, and using the `-J` to make it work (with the loud warning)